tests/installation/release_notes_from_url.pm

```diff
@@ -19,7 +19,13 @@ sub run {
     enter_cmd "zgrep -oh \"Got release notes.*\" /var/log/YaST2/y2log*";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag 'got-releasenotes-URL') {
-        record_soft_failure('bsc#1190711 - Release notes source does NOT match expectations or not found in YaST logs, expected source: URL');
+        my $not_public = script_run('grep -rE "GMC|RC" /README.BETA');
```

Comment:
there is public beta, this variable name might be confusing.

Reply:
the grep will return 0 when it's public and 1 when it's not public, therefore I chose the $not_public as a name.

Reply:
best practice is avoid negating the variables:
`my $is_public_release = script_run('!grep -rE "GMC|RC" /README.BETA');

Reply:
pardon me but that seems far from best practice :smiley: